Add notebook

kpenaramirez opened the pull request:
This ticket studies the available temperature sensors in the Simonyi Telescope during the ComCam campaign starting on October 17 2024 and ending on December 12 2024.

kpenaramirez committed:
SITCOM 2079 Temperature analysis sensor ComCam reviewed

bstalder reviewed (changes requested):
correction to requirement is any surface in the dome should be +1C/-2C with respect to the local ambient air temperature.

mostly same comments at SITCOM-1810.  ESS 112 should still be the target for the PCA, not 113.

kpenaramirez committed:
SITCOM-2079_Additional_comments_addressed

kpenaramirez commented:
Hi @bstalder, I addressed your comments. Please let me know if you need me to modify something else before being able to merge.
Thank you for your time.
Karla.

bstalder commented:
approved
